.github/workflows/rustdoc.yml

```diff
@@ -0,0 +1,28 @@
+# This Action triggers a build of Vector's Rustdoc each time a PR is merged
+# to `master`.
+#
+# Netlify account for Rustdoc site: https://app.netlify.com/sites/vector-rustdoc
+# Repo that builds the site: https://github.com/timberio/vector-rustdoc
+# Root URL for Rustdoc site: https://vector-rustdoc.netlify.app
+# Pretty URL for the published docs: https://rustdoc.vector.dev
+name: Publish Vector Rustdoc
+
+# Action is triggered only if changes are merged, not just pushed
+# See also the "if" statement in the job definition
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  trigger-site-build:
+    name: Trigger a build of Vector's Rustdoc site (hosted by Netlify)
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.merged == true # Trigger only on successful merge
+    steps:
+      - name: Trigger via CURL request
+        # https://github.com/wei/curl
+        uses: wei/curl@v1
```

Comment:
Could we tag this to a SHA ala the recommendations in https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions . Alternatively, I think you could also probably just use `curl` in a `run` step rather than a custom action.